yt/utilities/sdf.py

```diff
@@ -340,7 +340,7 @@ def write(self, filename):
                 k, v = var[0], _rev_types[var[1]]
                 to_write.append(k)
                 f.write(f"\t{v} {k};\n")
-            f.write("}[%i];\n" % s.size)
+            f.write(f"}}[{s.size}];\n")
```

Comment:
my brain is going fuzzy at this point... but pretty sure there's an extra } ? 
```suggestion
            f.write(f"}[{s.size}];\n")
```

Reply:
This is intended ! That how curly braces are escaped in fstrings

Reply:
ah! ok, great :)